### DISCUSSION TITLE
Powercord Theme Issues

### DISCUSSION THREAD

**Discussion by rondotdll** [janleigh/dotfiles#23] [Support] [answered]
I'm new to using powercord, so I may just be missing something but cloning the [powercord theme](https://github.com/janleigh/discord-css) into the `$HOME/powercord/src/Powercord/themes/` doesn't appear to be doing anything, and powercord is not detecting detecting it when manually refreshing the theme list.

**Comment by aishenreemo** [comment]
Did you run `make install` after cloning the repository?
```
cd $HOME/powercord/src/Powercord/themes/discord-css
make install
```

**Reply by rondotdll** [reply]
So that was the issue, but I thought I should tell you that by default cloning that repo clones from the rewrite branch (which doesn't have a make file) opposed to the master branch (which does have a make file). After checking out master and running make install however everything worked alright!

**Comment by rondotdll** [comment]
To install the powercord theme:

```
cd YOUR/POWERCORD/THEMES/FOLDER

git clone https://github.com/janleigh/discord-css
cd discord-css
git checkout master
make install
```

**Comment by janleigh** [comment, marked as answer]
Uhh... Kinda late but the master branch is no longer maintained. Here's how to compile the theme on the rewrite branch.
```sh
 sudo pacman -S nodejs # Cause we need yarn. Fuck you npm
 sudo corepack enable # Yarn is now shipped with nodejs >= 16.10
 
 cd YOUR/POWERCORD/THEMES/FOLDER
 git clone https://github.com/janleigh/discord-css && cd discord-css
 yarn build # Compile the theme
```
